selfservice/strategy/code/strategy_login.go

```diff
@@ -87,6 +87,9 @@ func (s *Strategy) HandleLoginError(r *http.Request, f *login.Flow, body *update
 }
 
 func (s *Strategy) PopulateLoginMethod(r *http.Request, requestedAAL identity.AuthenticatorAssuranceLevel, lf *login.Flow) error {
+	if requestedAAL > identity.AuthenticatorAssuranceLevel1 {
+		return nil
+	}
```

Comment:
Does this mean code cannot be used as second factor, or it even cannot be used as a first one when there is a second one?

Reply:
code isn't used in 2FA flows anyway atm. Right now this is a First factor only implementation and the nodes shouldn't render on a 2FA screen at all!